### DISCUSSION TITLE
The 2022 update is live!

### DISCUSSION THREAD

**Discussion by mdryden** [110yards/fantasy#143] [Announcements]
The 2022 season is upon us, and for once, 110 Yards is ready with time to spare!

Highlights for 2022:

* Commissioners can now renew a league in 2022.  Renewing a league will maintain all of your previous settings and automatically include all past managers.  After renewing, all commissioner settings are unlocked again, so you can make any necessary changes before starting the 2022 season (including removing any managers which are not playing this season).  If you renew your league, you'll also be able to see the results from any past seasons.
* Since this is the first time 110 Yards has ever started a season without a complete teardown of the previous site, you can actually see 2021 players stats and ranks during the draft this year.
* Discord integration for leagues which want to receive notifications from the site.  Notifications are configurable by the commissioner, and should be particularly useful for any leagues which do a remote draft over several days.
* The hard cap on limited positions (QB, RB, K) is now a commissioner option.  If the option is disabled, players in these positions will be allowed to be assigned to a bench position, and more than one starting spot can be created.  This option is disabled by default, and can be enabled on a per-position basis.  **For competitive balance reasons, I recommend keeping the hard caps in place, especially for larger leagues.**

Oh, and most importantly, I believe I have resolved the scoring related issues we had last year: players who lost their stats part way through the season, and players who accumulated stats but appeared in the player list twice (once with no team).

A full list of changes can be viewed here for anyone who is interested: https://github.com/mdryden/110yards/releases/tag/2022

2021 was a great season, and I'm excited to see what 2022 brings.  Good luck everyone!

**Comment by kamikazecanuck** [comment]
Awesome thanks!

**Comment by kamikazecanuck** [comment]
Looks like a lot of good improvements this year.

**Comment by bohsask** [comment]
Just a question regarding the 2021 player stats, I'm curious, are these based on:
a)Our league's 2021 scoring settings;
b) Our league's current scoring settings for 2022 if we changed them; 
c) The default settings.

**Comment by EElks** [comment]
We're excited to get started again, now that the CBA is a go.

We only have one suggestion--can we drop the email verification for signing it? It seems like overkill for a fun fantasy league--I'm not overly concerned about hackers who have more financially attractive sites to hit. I use a gmail account so it's not so bad, but those without one it is a royal pain in the butt.  Thanks for considering the change.

**Comment by MBControl-yyc** [comment]
Thanks for all your hard work.  We really appreciate it!

**Comment by riderfan2000** [comment]
Looks awesome.  First time user of the site.  Did our draft last night and went great!!!  Thanks so much we have been looking for something like this for a long time.  
I see you are asking for help....I'm not much of a website guy but I am good with spreadsheets.  Let me know if there is anything I can be a help with....would love to see this grow.

Cheers

**Comment by HoldenCaulfield23** [comment]
Just wanted to thank you for setting this up.  So far just completed the draft but I was rather impressed, even my not so tech savvy first time playing any fantasy football Dad was able to get it.  Updates have been great, really keeping up the crazy player movement in CFL.

Looking forward to the year.

**Comment by bohsask** [comment]
I was also wondering why some Americans have a maple leaf.  Darnell Sankey was another one I noticed.

**Comment by kamikazecanuck** [comment]
Maybe he’s been “Nationalized”. Lot of weird stuff in the new CBA.

**Comment by bohsask** [comment]
Naw, their status noted as 'Amer' on cfl.ca, I think we must have some kind of bug on this.
